packages/livecd/syslinux/build.yaml

```diff
@@ -9,14 +9,18 @@ prelude:
 {{else if .Values.is_fedora}}
 - dnf install -y wget
 {{end}}
-- wget http://kernel.org/pub/linux/utils/boot/syslinux/syslinux-{{.Values.version}}.tar.xz
-- mkdir -p /syslinux
-- tar -xvf syslinux-{{.Values.version}}.tar.xz -C /syslinux
+- |
+   PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
+   wget http://kernel.org/pub/linux/utils/boot/syslinux/syslinux-$PACKAGE_VERSION.tar.xz && \
+   mkdir -p /syslinux && \
+   tar -xvf syslinux-{{.Values.version}}.tar.xz -C /syslinux
 steps:
-- mkdir -p /output/boot/syslinux
-- mv /syslinux/syslinux-{{.Values.version}}/bios/core/isolinux.bin /output/boot/syslinux
-- mv /syslinux/syslinux-{{.Values.version}}/bios/com32/elflink/ldlinux/ldlinux.c32 /output/boot/syslinux
-- mv /syslinux/syslinux-{{.Values.version}}/bios/mbr/isohdpfx.bin /output/boot/syslinux
+- |
+   mkdir -p /output/boot/syslinux && \
+   PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
```

Comment:
This is the real point. The CI bumps the version by bumping the version number after `+` - the spec was just reading the version entirely. Now it strips everything after `+`, so it is safe to revbump it

Reply:
saves me the time asking why this was changed, nice!